config/routes.rb

```diff
@@ -223,6 +223,7 @@
 
   # Help
   get 'help' => 'help#faq', :as => 'help'
+  get 'help/:topic' => 'help#faq'
```

Comment:
@salbertson We were trying to disable Hound for ruby files because we don't have a code style for ruby yet. First we thought that Hound wouldn't check ruby files when not explicitly enabled but it looks like we were wrong. Now we tried to disable the checks (in [.hound.yml](https://github.com/diaspora/diaspora/blob/develop/.hound.yml#L6)) but that didn't work either.

We just want Hound to check our javascript files and maybe we will add other languages later. Even with the checks in js files we have some problems. We have our own [config file for jshint](https://github.com/diaspora/diaspora/blob/develop/config/.jshint.json) but Hound also checked things like single vs. double quotes in js files although we didn't enable that in our config. About an hour ago I [tried to disable those checks](https://github.com/diaspora/diaspora/commit/f27b9dbf797d02aa18076890241e139619714e50) by explicitly disabling the checks. According to the jshint docs there is no value `false` for `quotmarks` and `maxlen` but I don't know another way to tell Hound to stop those checks and I hope that this works now.